launch/joy.py

```diff
@@ -0,0 +1,16 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
```

Comment:
Just being curious, what's the advantage of generating the launch file via Python vs the "old" way?

Reply:
The big advantage is, that it is possible to use all Python features in the launch file. Anyway, the syntax of the launch file would make no difference for this application ;-)

Reply:
Good to know, I'll have to look into it more =) Thank you :bow: